app/controllers/users_controller.rb

```diff
@@ -1,16 +1,17 @@
 class UsersController < ApplicationController
 
   def show
+    
     @user = User.find_by(params[:user_id])
     if @user.token == nil
-      []
+      @repos = []
```

Comment:
This is what was causing us issues earlier

Reply:
Ahh nice catch.